packages/liblzo/include-lzo-lzodefs.h.patch

```diff
@@ -0,0 +1,12 @@
+--- a/include/lzo/lzodefs.h
++++ b/include/lzo/lzodefs.h
+@@ -2604,7 +2604,8 @@
+ #  define LZO_OPT_UNALIGNED64               1
+ #  endif
+ #elif (LZO_ARCH_ARM)
+-#  if defined(__ARM_FEATURE_UNALIGNED)
++#  if defined(__ANDROID__)
++#  elif 1 && defined(__ARM_FEATURE_UNALIGNED)
```

Comment:
Wouldn't `if defined(__ARM_FEATURE_UNALIGNED) && !defined(__ANDROID__)` be easier?

Reply:
That does not work and results in defining `LZO_OPT_UNALIGNED16` and `LZO_OPT_UNALIGNED32` as `1`.

What follows in the original source is shown below, with the "Boom!" comment added by me.

```c
#elif (LZO_ARCH_ARM)
#  if defined(__ARM_FEATURE_UNALIGNED)
#   if ((__ARM_FEATURE_UNALIGNED)+0)
#    ifndef LZO_OPT_UNALIGNED16
#    define LZO_OPT_UNALIGNED16             1
#    endif
#    ifndef LZO_OPT_UNALIGNED32
#    define LZO_OPT_UNALIGNED32             1
#    endif
#   endif
#  elif 1 && (LZO_ARCH_ARM_THUMB2)          /* Boom! */
#    ifndef LZO_OPT_UNALIGNED16
#    define LZO_OPT_UNALIGNED16             1
#    endif
#    ifndef LZO_OPT_UNALIGNED32
#    define LZO_OPT_UNALIGNED32             1
#    endif
#  elif 1 && defined(__ARM_ARCH) && ((__ARM_ARCH)+0 >= 7)
```

Reply:
Done.